test/test_regression_cli.py

```diff
@@ -46,12 +46,13 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir("./test/")
+    os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
added `os.chdir("./test/")`
but using a full path got from the file location
Note: we should avoid reference paths when ever possible. Sometimes the program just gets losts if one test errors